examples/toy_npe_model/GW150914.ini

```diff
@@ -0,0 +1,44 @@
+################################################################################
+##  Job submission arguments
+################################################################################
+
+local = True
+accounting = dingo
+request-cpus-importance-sampling = 16
+n-parallel = 4
+simple-submission = false
+
+################################################################################
+##  Sampler arguments
+################################################################################
+
+model = /path/to/model.pt
+device = 'cpu'
+num-gnpe-iterations = 30
+num-samples = 50000
+batch-size = 50000
```

Comment:
> Thanks for preparing this. I looked through the examples and made a number of suggestions, mostly pertaining to simplifying the toy model. I want to know also whether you have tested these examples, particularly the toy model?
> 
> I will look next at the documentation.

I've ran the toy example locally, but the problem I'm having is that with the settings I'm using the posterior comes out really bad. This is expected, but the problem is when plotting the posterior there is an error because of the importance weights.